src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -165,5 +166,14 @@
             <argument type="service" id="http_kernel" />
             <argument>%twig.exception_listener.controller%</argument>
         </service>
+
+        <service id="twig.configurator.environment" class="Symfony\Bundle\TwigBundle\DependencyInjection\Configurator\EnvironmentConfigurator">
```

Comment:
Please have a look at #13557 for a proposal about making it possible to use inlined configurator services in the XML format.